driver/config/config.go

```diff
@@ -1364,14 +1365,17 @@ func (p *Config) WebAuthnForPasswordless(ctx context.Context) bool {
 }
 
 func (p *Config) WebAuthnConfig(ctx context.Context) *webauthn.Config {
+	id := p.GetProvider(ctx).String(ViperKeyWebAuthnRPID)
+	origin := p.GetProvider(ctx).String(ViperKeyWebAuthnRPOrigin)
+	origins := p.GetProvider(ctx).StringsF(ViperKeyWebAuthnRPOrigins, []string{stringsx.Coalesce(origin, id)})
```

Comment:
The ID is not a URL iirc but instead the hostname so `ory.sh` for example. The origins however need to have the scheme. So I'm not sure if `id` is a good default.

Reply:
I took that behaviour from the config spec: "An explicit RP origin. If left empty, this defaults to `id`."

https://github.com/ory/kratos/pull/3380/files#diff-4a79b8650ca1bb0046229c6603ab7458e7711fc69812f13bbb8fc73b81b833e6R1601

In any case, origin is optional in the config but required for the WebAuthN config.

Reply:
OK SGTM